etc/schema.json

```diff
@@ -7865,6 +7872,7 @@
                 },
                 "pcap_log": {
                     "type": "object",
+                    "description": "Statistics for packet logging",
```

Comment:
I think `pcap` needs to be part of the description as the logging is specifically in the pcap format